src/routes/settings/Gift.tsx

```diff
@@ -198,12 +204,15 @@ export default function GiftPage() {
         return Number(getValue(giftForm, "amount")) < 50000;
     };
 
+    const selfHosted = state.mutiny_wallet?.get_network() === "signet";
```

Comment:
something is wrong here, either should check self hosted another way or rename this variable

Reply:
nice catch bad copy pasting by me